platform/net/ip/ip_resolver.go

```diff
@@ -18,8 +18,8 @@ func NetworkInterfaceToAddrsFunc(interfaceName string) ([]gonet.Addr, error) {
 }
 
 type Resolver interface {
-	// GetPrimaryIPv4 always returns error unless IPNet is found for given interface
-	GetPrimaryIPv4(interfaceName string) (*gonet.IPNet, error)
+	// GetPrimaryIP always returns error unless IPNet is found for given interface
+	GetPrimaryIP(interfaceName string, is_ipv6 bool) (*gonet.IPNet, error)
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... 

`is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)